blog/urls.py

```diff
@@ -15,9 +15,14 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from django.conf import settings
+from django.conf.urls.static import static
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     
     path('', include('feed.urls')),
-]
+   # path('<slug:slug>/', views.post_detail, name='post_detail')
```

Comment:
Comment not needed. That's why we use version control. If this is for reference, add a code comment

Reply:
for some reason, that code breaks the file